### PR TITLE
Support multiline text

### LINKED ISSUE #31: Add text with multiple lines
How to add text with multiple lines?

### PR DESCRIPTION
**Pull request by axion014**
Closes #31 and maybe #26.
I'm not entirely sure if this is needed though it probably does more good than bad.
I could also implement constant width auto wrapping but that's kinda costly performance-wise and difficult to do correctly. Distinct class for that, maybe?

### CODE REVIEW

File: src/SpriteText2D.ts [hunk 1]
```diff
@@ -42,15 +44,17 @@ export class SpriteText2D extends Text2D{
...
   }
 
+  get align() {
+    return this._align;
+  }
+
   set align(value: THREE.Vector2) {
     this._align.copy(value);
-    this._align.multiplyScalar(0.5);
-    this._align.addScalar(0.5);
```
Comment (axion014) [inline src/SpriteText2D.ts]: Sorry for sneaking in random fixes around here.

**Comment by endel** [comment]
Hi @axion014, thanks for the PR.

I've just tested this on my project, without making any change on it, and I'm having this error:

```
Uncaught TypeError: Cannot read property 'split' of null
    at CanvasText.drawText (CanvasText.js:26)
    at MeshText2D.updateText (MeshText2D.js:30)
    at MeshText2D.set [as text
```

**Comment by axion014** [comment]
Looks like you set null for the text parameter. previously it would showed "null". if that's what you want I'll add type coercion.

**Comment by endel** [comment]
Thanks @axion014, I've made a few changes to allow setting empty/null texts here, so this is not a breaking change for anyone 🤗  https://github.com/gamestdio/three-text2d/commit/938c950c1938b7f53f12cd79db0bc0404380cd2d

Thanks for your time on this PR!

**Pull request merged**